src/kayenta/service/canaryRun.service.ts

```diff
@@ -20,6 +26,17 @@ export const getCanaryRun = (configId: string, canaryExecutionId: string): Promi
       return run;
     });
 
+export const startCanaryRun = (
```

Comment:
I think you can send a very similar request to the `Tasks` endpoint to run a `kayentaCanary` stage.